### PR TITLE
obs-qsv11: Remove old MSDK 1.6 code

### PR DESCRIPTION
**Pull request by RytoEX**
### Description

The old MSDK 1.6 code paths were for older devices which we no longer support. Typically, version 1.0 is reported when QSV fails to initialize. The lowest version that should be supported is 1.35, so we should not need code that covers versions 1.0 to 1.6.

### Motivation and Context

Don't want dead code.

### How Has This Been Tested?

Tested locally on Windows 11 with an Arc A770M.

### Types of changes

- Code cleanup (non-breaking change which makes code smaller or more readable)

### Checklist:

- [x] My code has been run through [clang-format](https://github.com/obsproject/obs-studio/blob/master/.clang-format).
- [x] I have read the [**contributing** document](https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst).
- [x] My code is not on the master branch.
- [x] The code has been tested.
- [x] All commit messages are properly formatted and commits squashed where appropriate.
- [x] I have included updates to all appropriate documentation.

### CODE REVIEW

**Comment by RytoEX** [comment]
As a note, I commented out one or two lines of the frame type checking code in the first commit and then moved and uncommented it in the second commit. This is mostly to try to keep the original diff (the first commit) sane and readable, but both commits can be squashed if that makes more sense.

File: plugins/obs-qsv11/obs-qsv11.c [hunk 1]
```diff
@@ -1062,15 +1062,15 @@ static void parse_packet(struct obs_qsv *obsqsv, struct encoder_packet *packet,
...
-	//int64_t interval = obsqsv->params.nbFrames + 1;
-
 	packet->dts = ts_mfx_to_obs(pBS->DecodeTimeStamp, voi);
 
 #if 0
+	bool iFrame = pBS->FrameType & MFX_FRAMETYPE_I;
+	bool bFrame = pBS->FrameType & MFX_FRAMETYPE_B;
+	bool pFrame = pBS->FrameType & MFX_FRAMETYPE_P;
+	int iType = iFrame ? 0 : (bFrame ? 1 : (pFrame ? 2 : -1));
+	//int64_t interval = obsqsv->params.nbFrames + 1;
```
Comment (kkartaltepe) [inline plugins/obs-qsv11/obs-qsv11.c:1072]: can we just delete this interval?

**Review by kkartaltepe — APPROVED** [review]
Didnt test it, but it looks good.

File: plugins/obs-qsv11/obs-qsv11.c (hunk 1 above)
Comment (RytoEX) [reply plugins/obs-qsv11/obs-qsv11.c]: Removed this and another `interval` in another similar function.

**Pull request merged**